src/GraphQL/IDocumentWriter.cs

```diff
@@ -4,8 +4,14 @@
 
 namespace GraphQL
 {
+    /// <summary>
+    /// Serializes an object hiearchy to a stream. Typically this would be serializing an instance of the ExecutionResult class into a JSON stream.
```

Comment:
```suggestion
    /// Serializes an object hierarchy to a stream. Typically this would be serializing an instance of the ExecutionResult class into a JSON stream.
```